package.json

```diff
@@ -128,6 +128,5 @@
     ]
   },
   "dependencies": {
```

Comment:
We don't need this here so please remove it. Only `devDependencies` should be here.